cedar-policy/CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [3.1.0] - coming soon
+Cedar Language Version: 3.0.0
```

Comment:
I think this should be 3.1.0.  New schema syntax is a language minor

Reply:
Ack. Makes sense.